packages/document/doc-tools-doc/docs/en/plugin/official-plugins/overview.mdx

```diff
@@ -0,0 +1,5 @@
+# Overview
+
+Official plugins include the following:
+
+- [@edenx/doc-plugin-typedoc](./typedoc)：Integrate [TypeDoc](https://github.com/TypeStrong/typedoc), used to generate API documentation of TS module automatically.
```

Comment:
Should we add more documents for other doc plugins?

Reply:
I will add api-docgen plugin doc in the future.